projects/plugins/jetpack/modules/search/customberg/lib/analytics.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import {
+	initializeTracks,
+	identifySite as identifySiteTracks,
+	recordEvent as recordEventTracks,
+} from '../../instant-search/lib/tracks';
+
+// Adhere to following:<source>_<context>_<optional subcontext>_<action>_<optional qualifier>
+// (e.g. full event name: "jetpack_search_customberg_save_button_click").
+export const eventPrefix = 'jetpack_search_customberg';
+
+// Force enable analytics! Ignore isAnalyticsEnabled flag used by the Instant Search application.
+export const initialize = ( ...args ) => initializeTracks( ...args, true );
```

Comment:
tracking functionality could be extracted upper level, would be less coupled

Reply:
Could you clarify what you mean here? This module is merely a small wrapper for Instant Search's analytics wrapper.

Reply:
> Could you clarify what you mean here? This module is merely a small wrapper for Instant Search's analytics wrapper.

Was just thinking, analytics as a shared module would be better on the same level of instant search / customburg, rather than nested in instant search module. Definitely not a priority for now.

Reply:
Ah, got it.